build.gradle.kts

```diff
@@ -119,13 +111,12 @@ spotless {
 }
 
 java {
-    toolchain {
-        languageVersion.set(JavaLanguageVersion.of(21))
-    }
+    sourceCompatibility = JavaVersion.VERSION_21
+    targetCompatibility = JavaVersion.VERSION_21
```

Comment:
Why instead of toolchain ?
Both is possible and allowed

Reply:
just looks cleaner imo

Reply:
But both does different things

Reply:
just read about the difference and re-added it (but minified)